openedx/features/idea/models.py

```diff
@@ -2,5 +2,77 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import FileExtensionValidator
+from django_countries.fields import CountryField
 
-# Create your models here.
+from functools import partial
+
+from lms.djangoapps.onboarding.models import Organization
+from .helpers import upload_to_path
+
+
+class Location(models.Model):
+    country = CountryField()
+    city = models.CharField(max_length=255)
+
+    @property
+    def location(self):
+        return '{city}, {country}'.format(city=self.city, country=self.country)
+
+    def __unicode__(self):
+        return self.location
+
+    class Meta:
+        abstract = True
+
+
+class VisualAttachment(models.Model):
+
+    video_link = models.URLField(blank=True, null=True)
+    image = models.FileField(
```

Comment:
why arent we using image field? is it because of the 'pdf'? I dont think pdf is an image even though that is given in teh designs

Reply:
Yes just because of `pdf`, ImageField does not directly support `pdf`, some of supported extensions are listed [here](https://code.djangoproject.com/ticket/28242)

Reply:
I think they copied pdf there by mistake because we can't display pdf like how the image is displayed as well. And there would be extra handling required as well to handle it. I think we can convert it to image field and I'll get the design updated. (Also PDF is also supported the file field)
@ahmed-faraz46 @emraan004

Reply:
yes @zahid-arbisoft PDF option should not be images options